assets/ui.css

```
@@ -426,3 +426,22 @@
 #xref-ui:not(.ready) {
   background: url("https://respec.org/xref/loader.gif") no-repeat center;
 }
+
+.respec-dfn-list li span.exported, span.unused {
+  margin-left: 8px;
+  padding: 0.1em;
+  text-align: center;
+  white-space: nowrap;
+}
+
+.respec-dfn-list li span.exported {
+  background: hsl(205.7, 70%, 96.1%);
```

Nit: Lets round-off all the colors.
```suggestion
  background: hsl(206, 70%, 96%);
```

---

```suggestion
  background: #eef6fc;
```

---

changed the lot to hex in https://github.com/w3c/respec/pull/2711/commits/2e75a7b1b4d11d14de299f49fa1691915ddf1547